### PR TITLE
Instructions on using HTTPS

### LINKED ISSUE #452: instructions on using https
I just got CVAT working with HTTPs. I'm new to Django so I had to dig a bit through the gitter chat to find the info I needed. Would it be helpful for me to summarize the steps I took and add to instructions.md?

### PR DESCRIPTION
**Pull request by radoye**
Step-by-step instructions for https access to CVAT. No source changes.
Resolve #452

### CODE REVIEW

**Comment by radoye** [comment]
@nmanovic Sure, I'll clean up the .md to pass the linter.  We used this setup on multiple labeling servers, but it would be good if someone can confirm that it's working for them too.

**Comment by radoye** [comment]
@nmanovic Should be fixed now.

**Comment by nmanovic** [comment]
@radoye , Master branch is for stable release only. You need to choose develop branch to merge your changes.

File: cvat/apps/documentation/installation.md [hunk 1]
```diff
@@ -306,3 +307,299 @@ volumes:
...
+```bash
+server {
+    listen       80;
+    server_name  _ default;
+    return       404;
+}
+
+server {
+    listen       443 ssl;
+    server_name  ${CVAT_HOST};
```
Comment (azhavoro) [inline cvat/apps/documentation/installation.md:541]: Could you please also add 301 redirect to https:// for this server_name on 80 port?

**Comment by azhavoro** [comment]
> @azhavoro , I believe we already discussed the issue for our internal server. Can we avoid cloning of our configuration file? It will be a tough problem to support nginx config in several places.

I think it's possible, but I've not tested this approach yet.

**Comment by nmanovic** [comment]
@radoye , will you be able to fix the PR? Current problems:

- [ ] you need to merge into develop branch
- [ ] changelog isn't correct (the change should be for 1.0.0-beta)

After that we will be able to merge. @azhavoro will improve the documentation later (remove duplication of nginx config).

**Comment by radoye** [comment]
@nmanovic Yes, I was planning to do it by the end of the week. Sorry for the delay.

**Comment by radoye** [comment]
@nmanovic Changelog fixed. Target branch is now `opencv:develop`.

**Comment by radoye** [comment]
@nmanovic Should I rebase & resolve the CHANGELOG conflict or do you want to do it?

**Pull request merged**